analyzers/src/SonarAnalyzer.Common/Helpers/SonarLintXmlReader.cs

```diff
@@ -19,8 +19,6 @@
  */
 
 using System.IO;
-using System.Text;
```

Comment:
Why not doing this as part of Mary's PR just to keep the commit history one step smaller? She's changing the file already.

Reply:
She asked me to do it separately and yet if the commit history is such a concern, I will ask her to do it as part of her PR.